src/main/java/com/odysseusinc/arachne/datanode/service/client/atlas/AtlasClient2_7.java

```diff
@@ -0,0 +1,25 @@
+package com.odysseusinc.arachne.datanode.service.client.atlas;
+
+import com.odysseusinc.arachne.datanode.dto.atlas.EstimationListItem;
+import com.odysseusinc.arachne.datanode.dto.atlas.PatientLevelPredictionInfo;
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+import java.util.List;
+import org.springframework.http.MediaType;
+
+public interface AtlasClient2_7 extends AtlasClient {
+	@RequestLine("GET /prediction")
+	List<PatientLevelPredictionInfo> getPatientLevelPredictions();
+
+	@RequestLine("GET /prediction/{id}/download?packageName={package}")
```

Comment:
The same comment as [here](https://github.com/OHDSI/ArachneNodeAPI/pull/302#discussion_r278578972)

Reply:
Totally agree, this should be reviewed and merged with new CC and Pathways imports